charts/gardener/gardenlet/values.yaml

```diff
@@ -79,9 +79,7 @@ global:
         # - production
         seed:
           concurrentSyncs: 5
-        # TODO (plkokanov): the sync period is currently set to 1m because of the way that SNI detection is done during seed reconciliations.
-        # ref: https://github.com/gardener/gardener/issues/6036
-          syncPeriod: 1m
+          syncPeriod: 15m
```

Comment:
How you came up with the value of `syncPeriod` as `15m` ?

Reply:
I haven't deduced it, I took the value from this [PR](https://github.com/gardener/gardener/pull/5995).

Reply:
We would have used `1h` in the PR that you linked: https://github.com/gardener/gardener/pull/5995#discussion_r879086695